netket/utils/config_flags.py

```diff
@@ -126,6 +126,19 @@ def update(self, name, value):
     runtime=False,
 )
 
+config.define(
+    "NETKET_MPI",
+    bool,
+    default=True,
+    help=dedent(
+        """
+        Used to forcibly shut-down MPI in NetKet. If this flag is
```

Comment:
```suggestion
        Prevent NetKet from using (and initializing) MPI. If this flag is
```

Reply:
we should hook those docstrings up with the website eventually... or have some way to print them.